client/landing/gutenboarding/onboarding-block/style.scss

```diff
@@ -10,6 +13,16 @@
 	top: 20%;
 	left: 0%;
 	width: 100%;
+
+	.onboarding-block__question-answered,
+	.onboarding-block__question-input,
+	.onboarding-block__question-input::placeholder,
+	.onboarding-block__question,
+	.onboarding-block__questions-heading,
+	.onboarding-block__question-skip {
+		transition: $onboarding-block-css-transition-duration color linear
```

Comment:
Transitioning `color` is expensive for the UI and a possible alternative approach would be to wrap the block content with `SwitchTransition` and `CSSTransition` (instead of just the background) and use the classes provided to transition opacity. But since this text color transition is needed just for the dark mode, I would wait for the design feedback first and add just a @TODO for now.